haiku-apps/exrtranslator/exrtranslator-0.1.0.recipe

```diff
@@ -0,0 +1,81 @@
+SUMMARY="EXR Translator addon for Haiku"
+DESCRIPTION="EXR Translator is a translator add-on for displaying EXR images.
+
+Using this add-on will allow any application using Haiku Data Translations \
+to view the EXR file format files."
+HOMEPAGE="https://github.com/HaikuArchives/EXRTranslator"
+COPYRIGHT="2008 Haiku Inc."
+LICENSE="MIT"
+REVISION="1"
+if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+	srcGitRev="6cdbb0b924cbcdeb85c6d55c95b244cc0353c312"
+	CHECKSUM_SHA256="676bb92c660ef30e622f787fa36aee746ce8788e2c7cda3dbc9de9b34eab89df"
+else
+	srcGitRev="62856f5b5b797d00ff1356519ab5ce1c428ea660"
+	CHECKSUM_SHA256="66752ac5ad4307b285ae6255a3624dce9a98414ed3c00ba1c87a4007a8005ee8"
+fi
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+SOURCE_DIR="EXRTranslator-$srcGitRev/src"
+
+ARCHITECTURES="all"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	exrtranslator$secondaryArchSuffix = $portVersion
+	add_on:EXRTranslator$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+	REQUIRES+="
+		lib:libHalf$secondaryArchSuffix
+		lib:libIex_2_2$secondaryArchSuffix
+		lib:libIexMath_2_2$secondaryArchSuffix
+		lib:libIlmImf_2_2$secondaryArchSuffix
+		lib:libIlmImfUtil_2_2$secondaryArchSuffix
+		lib:libIlmThread_2_2$secondaryArchSuffix
+		lib:libImath_2_2$secondaryArchSuffix
+		"
+else
+	REQUIRES+="
+		lib:libImath_3_1$secondaryArchSuffix
+		lib:libOpenEXR_3_2$secondaryArchSuffix
+		"
+fi
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+if [ "$effectiveTargetArchitecture" = x86_gcc2 ]; then
+	BUILD_REQUIRES+="
+		devel:libHalf$secondaryArchSuffix
+		devel:libIlmImf_2_2$secondaryArchSuffix
+		devel:libIlmImfUtil_2_2$secondaryArchSuffix
+		"
+else
+	BUILD_REQUIRES+="
+		devel:libImath_3_1$secondaryArchSuffix
+		devel:libOpenEXR_3_2$secondaryArchSuffix
+		"
+fi
+
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	"
+
+BUILD()
+{
+	if [ -n "$secondaryArchSuffix" ]; then
+		sed -i 's|develop/headers/|develop/headers/x86/|g' Makefile
```

Comment:
We currently have only one kind of secondaryArch, so it doesn't matter, but a generic replacement would be better I think:

```suggestion
		sed -i "s|develop/headers/|develop/headers$secondaryArchSubDir/|g" Makefile
```

Reply:
Can't commit here, doing a check on 32bit and will do a revbump then. 
Pokes @korli in case you are watching this.

Reply:
secondary arches aren't "all" but only x86, so in this case it doesn't matter.

Reply:
OK, then we can leave it as is. Thanks for responding!